pymysql/cursors.py

```diff
@@ -93,14 +94,32 @@ def _nextset(self, unbuffered=False):
     def nextset(self):
         return self._nextset(False)
 
+    def _ensure_bytes(self, x, encoding=None):
+        if not PY2:
+            return x
```

Comment:
It's not required, as all the calling sites are protected by a `if PY2` check. I just put it there as the next line has `unicode` on it, which isn't even defined in Python 3, so it's more of a precaution if someone else where to refactor the code and call this from somewhere else. I'd be happy to remove the check here.